css-syntax-3/Overview.bs

```diff
@@ -789,15 +789,15 @@ Definitions</h3>
 		<dd>
 			A <a>code point</a> with a value equal to or greater than U+0080 &lt;control>.
 
-		<dt><dfn export lt="identifier-start code point | name-start code point" oldids="name-start-code-point">identifier-start code point</dfn>
```

Comment:
This needs to stay, so old references continue to link.

Reply:
Sure thing! This is done in 7103a9f.

Should I also (or alternatively) add anything to `oldids`?